index.html

```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
         <li><i class="fa-brands fa-facebook-f"></i></li>
       </ul>
     </section>
+    <section class="card">
```

Comment:
Great work was done so far 👍🏼  , I want just to mention that the background color you used for this section is diffrent from the template kindly fixe it 

![image (34)](https://github.com/enocol/MobileFirst/assets/59707859/827e1be6-cc40-4b40-a59f-d85ec38e7c71)

![image (33)](https://github.com/enocol/MobileFirst/assets/59707859/66d5820a-0ae3-48e6-977d-ab2c7e82dae3)